pkg/openshift/filesystem/filesystem_test.go

```diff
@@ -0,0 +1,43 @@
+package filesystem
+
+import (
+	"os"
+	"testing"
+)
+
+type faketemplates struct{}
```

Comment:
parts of this test need to be melded into the other, and this file goes away as you move GetFileInfo into pkg/certgen